Source/WebKit/GPUProcess/media/RemoteLegacyCDMSessionProxy.cpp

```diff
@@ -80,11 +83,12 @@ static RefPtr<WebCore::SharedBuffer> convertToOptionalSharedBuffer(T array)
 
 void RemoteLegacyCDMSessionProxy::generateKeyRequest(const String& mimeType, RefPtr<SharedBuffer>&& initData, GenerateKeyCallback&& completion)
 {
+    if (!m_session)
+        return completion({ }, emptyString(), 0, 0);
```

Comment:
There was a long thread on webkit-dev about the appropriateness of `return void;`. I think the project as a whole decided against allowing this.

Reply:
Sounds good